shortcut_composer/actions.action

```diff
@@ -101,6 +101,10 @@
         <activationFlags>1</activationFlags>
         </Action>
 
+        <Action name="Pick Color Sampler blend (%)">
```

Comment:
Ten (%) chyba nie jest potrzebny. Wybierasz "Sampler blend", a to jaki tam będzie do tego interface nie jest na tym etapie istotny.